docs/setup/import.md

```diff
@@ -48,12 +48,12 @@ Once the import is completed, you can merge the changes from the import branch t
 #### Prerequisite
 
 lakeFS must have permissions to list the objects at the source object store,
-and on the same region of your destination bucket.
+and in the same region of your destination bucket.
 
 #### Limitations
 
 Import is feasible only from source object storage that matches the storage namespace of the
-current repository.  lakeFS S3 installation cannot import from Azure bucket.
+current repository. The lakeFS S3 Installation cannot import from Azure bucket.
```

Comment:
This sentence should be:

_Importing is only possible from the object storage service in which your installation stores its data. For example, if lakeFS is configured on top of S3, you cannot import data from Azure._